tests/snuba/api/endpoints/test_project_event_details.py

```diff
@@ -115,6 +119,160 @@ def test_ignores_different_group(self):
         assert response.data["nextEventID"] is None
 
 
+class ProjectEventDetailsTransactionTest(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super().setUp()
+        self.login_as(user=self.user)
+        project = self.create_project()
+
+        one_min_ago = iso_format(before_now(minutes=1))
+        two_min_ago = iso_format(before_now(minutes=2))
+        three_min_ago = iso_format(before_now(minutes=3))
+        four_min_ago = iso_format(before_now(minutes=4))
+
+        transaction_event_data = {
+            "level": "info",
+            "message": "ayoo",
+            "type": "transaction",
+            "culprit": "app/components/events/eventEntries in map",
+            "contexts": {"trace": {"trace_id": "b" * 32, "span_id": "c" * 16, "op": ""}},
+        }
+
+        def hack_pull_out_data(jobs, projects):
+            _pull_out_data(jobs, projects)
+            for job in jobs:
+                job["event"].groups = [self.group]
+            return jobs, projects
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            self.prev_transaction_event = self.store_event(
+                data={
+                    **transaction_event_data,
+                    "event_id": "a" * 32,
+                    "timestamp": four_min_ago,
+                    "start_timestamp": four_min_ago,
+                },
+                project_id=project.id,
+            )
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            self.cur_transaction_event = self.store_event(
+                data={
+                    **transaction_event_data,
+                    "event_id": "b" * 32,
+                    "timestamp": three_min_ago,
+                    "start_timestamp": three_min_ago,
+                },
+                project_id=project.id,
+            )
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            self.next_transaction_event = self.store_event(
+                data={
+                    **transaction_event_data,
+                    "event_id": "c" * 32,
+                    "timestamp": two_min_ago,
+                    "start_timestamp": two_min_ago,
+                    "environment": "production",
+                    "tags": {"environment": "production"},
+                },
+                project_id=project.id,
+            )
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
```

Comment:
I think that we don't want to have this in place. That way this transaction doesn't have the group id associated with it.